docker/build-tools/build-and-push.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 HUB=gcr.io/istio-testing
-VERSION=$(date +%Y-%m-%d)
+VERSION=$(date +%Y-%m-%dT%H-%M-%S)
```

Comment:
it's nice to have the git sha

Reply:
ya I guess, although its harder to use a time stamp.